internal/pkg/installer/manifest/manifest.go

```diff
@@ -112,29 +111,12 @@ func NewManifest(install machine.Install) (manifest *Manifest, err error) {
 		manifest.Targets[target.Device] = append(manifest.Targets[target.Device], target)
 	}
 
-	for _, extra := range install.ExtraDisks() {
-		if manifest.Targets[extra.Device] == nil {
-			manifest.Targets[extra.Device] = []*Target{}
-		}
-
-		for _, part := range extra.Partitions {
-			extraTarget := &Target{
-				Device: extra.Device,
-				Size:   part.Size,
-				Force:  true,
-				Test:   false,
-			}
-
-			manifest.Targets[extra.Device] = append(manifest.Targets[extra.Device], extraTarget)
-		}
-	}
-
```

Comment:
For some reason, we had extra disks as part of the installation, but it shouldn't be. The extra disks should stand completely independent of the installation instructions.